code/DebugBarDatabaseCollector.php

```diff
@@ -34,12 +34,13 @@ public function collect()
         // Check for excessive number of queries
         $dbQueryWarningLevel = DebugBar::config()->warn_query_limit;
         if ($dbQueryWarningLevel && $data['nb_statements'] > $dbQueryWarningLevel) {
+            $helpLink = DebugBar::config()->performance_guide_link;
```

Comment:
I'm well aware of the magic getters and setters. I disagree it's good/fine practice to do so. As it's worth a lot to all SilverStripe devs, using best practices over "it works, it's fine" should be applied.